docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
+
+## Mac and Linux
+
+1. Determine the location of your certificates:
+    ```python
+    >>> import certifi
+    >>> certifi.where()
+    '/path/to/site-packages/certifi/cacert.pem'
+    ```
+2. Copy your custom self-signed certificate and paste into the
+`cacert.pem` file.
+
+## Windows
+
+1. Press «Win + R» to open a new command prompt and type `certmgr`.
+This opens the Windows Certificate Manager for the current user.
+2. Search all certificates stores for a certificate issued by `mitm`.
+There should be at least one certificate.
+
+   ![MITM certificate](../imgs/certmgr-windows.png)
+
+3. Export the certificate in Base-64 encoded X.509 (.CER) to your
+file system (`Path\To\mitm.cer`)
+4. Add the certificate to the list of trusted CA's via the
+`REQUESTS_CA_BUNDLE` environment variable:
+    ```bash
+    > set REQUESTS_CA_BUNDLE=Path\To\mitm.cer
+    ```
```

Comment:
This will make _only_ the custom certificate work. If some customer's VPN only hijacks some websites and not others, then they'll need to trust both the original certifi certificates and their custom one. They'll need to concat the two files, etc.:

```
type ...\custom.cer AppData\Roaming\Python\Python311\site-packages\certifi\cacert.pem > ...\everything.pem
```

where `AppData\Roaming\Python\Python311\site-packages\certifi\cacert.pem` was the output of `certifi.where()`, and then use `REQUESTS_CA_BUNDLE=...\everything.pem`

Reply:
Updated as follows to prepend the custom cert:

Mac: `export REQUESTS_CA_BUNDLE=/path/to/converted/certificate.crt:$REQUESTS_CA_BUNDLE`
Linux: `export REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt:$REQUESTS_CA_BUNDLE`
Windows: `set REQUESTS_CA_BUNDLE=%REQUESTS_CA_BUNDLE%;Path\To\mycert.cer`

Reply:
This won't actually prepend anything:
- It will only include the previous value of the environment variable, which is almost certainly unset (if the user had already set it to something, he/she probably wouldn't need these instructions)
- The docs don't even mention that the variable allows multiple files, so it probably doesn't. Pretty sure all certs have to be concatenated into a single file.

So on Windows, need to run the `type` command, or on Mac, it should be `cat`.

Reply:
Simplified: removed the whole prepend commands and just keep the `export` command.